lib/private/legacy/OC_App.php

```diff
@@ -295,51 +167,16 @@ public static function registerAutoloading(string $app, string $path, bool $forc
 		}
 	}
 
-	/**
-	 * Load app.php from the given app
-	 *
-	 * @param string $app app name
-	 * @throws Error
-	 */
-	private static function requireAppFile(string $app) {
-		// encapsulated here to avoid variable scope conflicts
-		require_once $app . '/appinfo/app.php';
-	}
-
 	/**
 	 * check if an app is of a specific type
 	 *
 	 * @param string $app
 	 * @param array $types
 	 * @return bool
+	 * @deprecated 26.0.0 call \OC::$server->get(IAppManager::class)->isType($app, $types)
```

Comment:
```suggestion
	 * @deprecated 26.0.0 use IAppManager::isType
```

to not encourage service locators